controllers/datafoundation_client.go

```diff
@@ -0,0 +1,134 @@
+package controllers
+
+import (
+	"fmt"
+
+	v1alpha1 "github.com/red-hat-storage/managed-fusion-agent/api/v1alpha1"
+	"github.com/red-hat-storage/managed-fusion-agent/datafoundationclient"
+	ocsclient "github.com/red-hat-storage/ocs-client-operator/api/v1alpha1"
+	ocsv1alpha1 "github.com/red-hat-storage/ocs-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+)
+
+type dataFoundationClientSpec struct {
+	onboardingTicket  string
+	storageDFEndpoint string
+}
+
+type dataFoundationClientReconciler struct {
+	*ManagedFusionOfferingReconciler
+
+	offering      *v1alpha1.ManagedFusionOffering
+	spec          dataFoundationClientSpec
```

Comment:
Is this a good name for a variable in the reconciler?

Reply:
updated